src/ddocs/ddocs.rst

```diff
@@ -116,17 +116,25 @@ single value - which could be an array or similar object.
 Built-in Reduce Functions
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Additionally, CouchDB has three built-in reduce functions. These are implemented
-in Erlang and run inside CouchDB, so they are much faster than the equivalent
-JavaScript functions: ``_sum``, ``_count`` and ``_stats``. Their equivalents in
-JavaScript:
+Additionally, CouchDB has a set of built-in reduce functions. These are
+implemented in Erlang and run inside CouchDB, so they are much faster than the
+equivalent JavaScript functions.
 
-.. code-block:: javascript
+.. data:: _approx_count_distinct
 
-    // could be replaced by _sum
-    function(keys, values) {
-        return sum(values);
-    }
+Since version `2.2.0`.
```

Comment:
I think we can use the `versionadded` markup for this:

    .. versionadded:: 2.2.0

Reply:
Ah, I should have grep'ed around for that. Will add, but set to `2.2` to maintain consistency with other usage in the docs.